docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -1,60 +1,175 @@
 [[getting-started-apm-server]]
-== Getting started with APM Server
+== Get started with APM Server (self managed)
```

Comment:
_Quick start with Elastic Cloud_ versus _Get started (Self managed)_ in the navigation bar doesn't seem too coherent. Also not a great suggestion, but wdyt about something along the lines of:
- Quick start with Elastic Cloud
- Self manage APM Server

Reply:
I do like this better. Let me think about it for a few mins.